reporting_test.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+
+from unittest.mock import patch
+
+import os
+import pyfakefs
+import unittest
```

Comment:
I always use pytest.

Reply:
Hmm, OK, added an issue to consider switching to pytest.